service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java

```diff
@@ -61,7 +63,7 @@ public void sendOrderByPost(final GeneratedOrderEvent orderEvent) {
 
     private void sendNotificationToEmailServedRepresentatives(final CaseData caseData,
                                                               final DocumentReference orderDocument) {
-        List<Representative> emailRepresentatives = caseData.getRepresentativesByServedPreference(EMAIL);
+        Set<String> emailRepresentatives = representativesInbox.getEmailsByPreference(caseData, EMAIL);
 
         if (!emailRepresentatives.isEmpty()) {
```

Comment:
@samuelzerozero should there be a similar set of changes for [SendNoticeOfHearingHandler](https://github.com/hmcts/fpl-ccd-configuration/blob/master/service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/SendNoticeOfHearingHandler.java) to cover the 3rd AC for
- Notice of New Hearing

Reply:
?

Reply:
`  
@Async
    @EventListener
    public void notifyRepresentatives(final SendNoticeOfHearing event) {
        final CaseData caseData = event.getCaseData();

        SERVING_PREFERENCES.forEach(servingPreference -> {
            NotifyData notifyData =
                newHearingContent.buildNewNoticeOfHearingNotification(
                    caseData, event.getSelectedHearing(), servingPreference);

            representativeNotificationService
                .sendToRepresentativesByServedPreference(
                    servingPreference, NOTICE_OF_NEW_HEARING, notifyData, caseData);
        });
    }
`
representativeNotificationService.sendToRepresentativesByServedPreference has been already modified to gather the extra requirement https://github.com/hmcts/fpl-ccd-configuration/blob/9ebcfdeb0b4c8bc919b9011fc4238793c1407a94/service/src/main/java/uk/gov/hmcts/reform/fpl/service/representative/RepresentativeNotificationService.java#L25

Reply:
cheers for explaining @samuelzerozero